docs/workspace/index.md

```diff
@@ -0,0 +1,31 @@
+# Workspaces
```

Comment:
Since this will be shown on the live documentation site once merged, I think we should put a big "this is being built and is not yet ready for use" disclaimer at the top to prevent "why doesn't this work yet?" questions.

Reply:
@nwalters512 it will be hidden from being served as it wasn't added to `mkdocs.yml`. 

https://github.com/PrairieLearn/PrairieLearn/blob/a4f35b5560305ae3b0051bfbeafcf59be170dd04/mkdocs.yml#L32

That said, @tdy you probably want to add it to the `mkdocs.yml` configuration and add a warning.

Reply:
Good point, @coatless. Maybe we should actually leave it hidden for now?

Reply:
> @nwalters512 it will be hidden from being served as it wasn't added to `mkdocs.yml`.

Good point!